reactioncommerce/reaction
# Update read only Quantity in top variant #1919

Resolves #1919 

**Description**
When a variant has second level variants the edit variant screen still shows the most recent quantity of top-level variant products, not updating it to use the combination of second-level variants, which is the true inventory count.

**Test**
1.  Create product.
2.  Create child variants and add quantities.
3. View edit variant section. On the top variant quantity expect to see total quantity of the child variants.
